src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -224,6 +253,14 @@ class FeatureSearchStore implements IFeatureSearchStore {
                     );
                 });
 
+                if (featureLifecycleEnabled) {
+                    query.leftJoin(
```

Comment:
I would not join it into original query, but rather into final query, where the paging has been done already.